newdevtech/newdev_api
# #166172699 Setup database migration

#### What does this PR do?
- Sets up the database migration files

#### Description of Task to be completed?
- add migration files for database models
- add role column to the user table
- add database config
- add scripts for db migrations
- add debug module to package.json
- setup server.js
- setup app.js
- add `sequelizerc` file
- delete webstorm code style files
- add `.env.example` file
- add `.babelrc`

#### How should this be manually tested?
- Having set up your local repository:
  - add `.env` file and variables from `.env.example`
  - run `npm run db:migrate`
  - confirm that the migration files have been run on your local computer

#### What are the relevant pivotal tracker stories?
[#166172699](https://www.pivotaltracker.com/story/show/166172699)

#### Screenshots?
<img width="1434" alt="Screen Shot 2019-05-26 at 8 09 18 PM" src="https://user-images.githubusercontent.com/43746609/58386050-3fc1f600-7ff2-11e9-83a4-2c1a61ef4e49.png">
